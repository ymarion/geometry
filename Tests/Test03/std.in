# Creating a circle
C C1 0 0 5
# Should show C1
LIST

