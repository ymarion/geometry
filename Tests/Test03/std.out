# Creation
OK
# LIST
C C1 0 0 5
